scss/_dark-variables.scss

```diff
@@ -71,168 +73,172 @@ $theme-colors: map-merge(
-$toast-header-background-color:     rgba($black, .85);
-$toast-header-border-color:         rgba(255, 255, 255, .05);
+$toast-header-color:                $gray-400 !default;
+$toast-header-background-color:     rgba($black, .85) !default;
+$toast-header-border-color:         rgba(255, 255, 255, .05) !default;
 
 // Dropdowns
 //
-$dropdown-color:                    $body-color;
-$dropdown-bg:                       $black;
-$dropdown-border-color:             rgba($white, .15);
-$dropdown-divider-bg:               $gray-800;
-$dropdown-box-shadow:               0 .5rem 1rem rgba($white, .175);
+$dropdown-color:                    $body-color !default;
+$dropdown-bg:                       $black !default;
+$dropdown-border-color:             rgba($white, .15) !default;
+$dropdown-divider-bg:               $gray-800 !default;
+$dropdown-box-shadow:               0 .5rem 1rem rgba($white, .175) !default;
 
-$dropdown-link-color:               $gray-100;
-$dropdown-link-hover-color:         lighten($gray-100, 5%);
-$dropdown-link-hover-bg:            $gray-900;
+$dropdown-link-color:               $gray-100 !default;
+$dropdown-link-hover-color:         lighten($gray-100, 5%) !default;
+$dropdown-link-hover-bg:            $gray-900 !default;
 
-$dropdown-link-active-color:        $component-active-color;
-$dropdown-link-active-bg:           $component-active-bg;
+$dropdown-link-active-color:        $component-active-color !default;
+$dropdown-link-active-bg:           $component-active-bg !default;
 
-$dropdown-link-disabled-color:      $gray-400;
+$dropdown-link-disabled-color:      $gray-400 !default;
 
-$dropdown-header-color:             $gray-400;
+$dropdown-header-color:             $gray-400 !default;
 
 // Breadcrumbs
-$breadcrumb-bg:                     $gray-800;
-$breadcrumb-divider-color:          $gray-400;
-$breadcrumb-active-color:           $gray-400;
+$breadcrumb-bg:                     $gray-800 !default;
+$breadcrumb-divider-color:          $gray-400 !default;
+$breadcrumb-active-color:           $gray-400 !default;
 
 // Pagination
-$pagination-color:                  $link-color;
-$pagination-bg:                     $black;
-$pagination-border-color:           $gray-700;
-
-$pagination-focus-box-shadow:       $input-btn-focus-box-shadow;
-
-$pagination-hover-color:            $link-hover-color;
-$pagination-hover-bg:               $gray-800;
-$pagination-hover-border-color:     $gray-700;
-
-$pagination-active-color:           $component-active-color;
-$pagination-active-bg:              $component-active-bg;
-$pagination-active-border-color:    $pagination-active-bg;
-
-$pagination-disabled-color:         $gray-400;
-$pagination-disabled-bg:            $black;
-$pagination-disabled-border-color:  $gray-700;
+$pagination-color:                  $link-color !default;
+$pagination-bg:                     $black !default;
+$pagination-border-color:           $gray-700 !default;
+
+$input-btn-focus-width:             .2rem !default;
```

Comment:
Good catch on those missing `$input-btn-focus-*` variables. However, is there a reason for `$input-btn-focus-width`?